src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/jira/StandaloneFormula.kt

```diff
@@ -180,6 +181,7 @@ class StandaloneFormula private constructor(
         CloseableThreadContext.push("Jira node").use {
             key.get().file.facilitateSsh(jiraIp)
         }
+        val isMySql = database is MySqlDatabase
```

Comment:
This will cause `Database` decorators to malfunction. [They do exist in the wild](https://github.com/atlassian/jira-hardware-exploration/blob/a8395bec1f96a36a4a5f4e03527e071532c932ae/src/test/kotlin/com/atlassian/performance/tools/lib/infrastructure/ConfigurableMysqlDatabase.kt).
In that case, `isMySql` will be `false` and subsequently we'll delete `dbconfig.xml` and break that use case.

So let's detect `is PostgresDatabase` instead, because such callers don't exist yet. This is a short-term hack, so we can still 🤞 for the [infra/aws-infra fix](https://github.com/atlassian/infrastructure/pull/43).

- [ ] detect `is PostgresDatabase` instead